src/main.rs

```diff
@@ -86,3 +86,45 @@ fn print_verbose_err() {
          https://github.com/rust-lang/rust-bindgen/issues/new"
     );
 }
+
+#[cfg(test)]
+mod test {
+    use std::path::PathBuf;
+
+    fn build_flags_output_helper(builder: &bindgen::Builder) {
+        let mut command_line_flags = builder.command_line_flags();
+        command_line_flags.insert(0, "bindgen".to_string());
+
+        let flags_quoted: Vec<String> = command_line_flags
+            .iter()
+            .map(|x| format!("'{}'", x))
+            .collect();
+        let flags_str = flags_quoted.join(" ");
+        println!("{}", flags_str);
+
+        crate::options::builder_from_flags(command_line_flags.into_iter())
+            .unwrap();
+    }
+
+    #[test]
+    fn commandline_flag_roundtrip() {
+        // test1: various options
```

Comment:
I _think_ #1821 is a better approach for this test, but...

Reply:
removed